pyproject.toml

```diff
@@ -116,6 +116,7 @@ tests-only = [
   "pytest-mpl>=0.16",
   "pytest-xdist>=3.0.2",
   "pytest>=7.1.0",
+  "itables>=2.2.4",
```

Comment:
I wonder if it would be worth combining this with `ipywidgets` (an optional dependency for parfive) in a `jupyter` extra we don't include in `all`?